project_event/models/project_task.py

```diff
@@ -94,3 +99,13 @@ def name_search(self, name='', args=None, operator='ilike', limit=100):
             domain = ['|', ('name', operator, name),
                       ('code', operator, name)]
         return super(Task, self).search(domain + args, limit=limit).name_get()
+
+    @api.one
+    def _compute_order_task(self):
+        fmt = '%Y-%m-%d %H:%M:%S'
+        if self.parent_id != False:
```

Comment:
@jananjoy simplify this expression

Reply:
@rimbendhaou Need for the precise explanation. Is it on format expression or condition checking.

Reply:
@jananjoy if self.parent_id